### DISCUSSION TITLE
Vásárlói kijelző más eszközön

### DISCUSSION THREAD

**Discussion by laurelkft** [nav-gov-hu/eRECEIPT#5] [Kérdések és válaszok]
Több PC-s környezetben előfordul, hogy más eszközök is vannak a kasszazónában. Akár 9"-os grafikus kijelzővel rendelkező terminálok is.

Lehetséges-e, hogy az új szabályzásban a vásárlói tájékoztatást egy teljesen szeparált eszközön tegyük meg. (Banki terminál)
A pénztárgépen ugyanúgy be van állítva a vásárlói kijelző, ugyanúgy kommunikál vele, és ugyanúgy képes ellenőrizni a jelenlétet, de a kijelző nem a kaszára van csatlakoztatva, hanem egy teljesen másik eszköz, akár önálló OS-sel.

**Comment by renced42** [comment]
Kedves @laurelkft 
A pénztárgép megoldásokhoz különböző perifériák csatlakoztathatók. Kérdés, hogy ez a külön kijelző milye módon része az engedélyezendö pénztárgépnek és milyen módon csatlakozik ahhoz.
